src/widget/wtracktableview.cpp

```diff
@@ -904,6 +947,34 @@ void WTrackTableView::setSelectedTracks(const QList<TrackId>& trackIds) {
     }
 }
 
+bool WTrackTableView::setCurrentTrackId(const TrackId& trackId, const int column = 0) {
```

Comment:
Default argument in implementation? Please also remove the `const` keyword.

Reply:
Oh. Fixed